.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
- - 5.5
```

Comment:
PHPunit 5.2 does not support PHP  5.5 anymore. That's ok for running the tests IMO.